packages/components/src/button/stories/index.js

```diff
@@ -128,6 +128,36 @@ Default.args = {
 	children: 'Code is poetry',
 };
 
+export const Primary = Template.bind( {} );
+Primary.args = {
+	...Default.args,
+	variant: 'primary',
+};
+
+export const Secondary = Template.bind( {} );
+Secondary.args = {
+	...Default.args,
+	variant: 'secondary',
+};
+
+export const Tertiary = Template.bind( {} );
+Tertiary.args = {
+	...Default.args,
+	variant: 'tertiary',
+};
+
+export const Link = Template.bind( {} );
+Link.args = {
+	...Default.args,
+	variant: 'link',
+};
+
+export const IsDestructive = Template.bind( {} );
+IsDestructive.args = {
+	...Default.args,
+	isDestructive: true,
+};
+
```

Comment:
Personally, yes. The guiding principle for me has been to highlight the main use cases as stories, since they can often be hard to discover from just the props table when you're not familiar with a component.